samples/snippets/src/main/java/com/example/analytics/QuickstartSample.java

```diff
@@ -16,82 +16,104 @@
 
 package com.example.analytics;
 
-/* This application demonstrates the usage of the Analytics Data API using
- service account credentials. For more information on service accounts, see
- https://cloud.google.com/iam/docs/understanding-service-accounts
+/* Google Analytics Data API sample quickstart application.
 
- The following document provides instructions on setting service account
- credentials for your application:
- https://cloud.google.com/docs/authentication/production
+This application demonstrates the usage of the Analytics Data API using service account credentials.
 
- In a nutshell, you need to:
+Before you start the application, please review the comments starting with
+"TODO(developer)" and update the code to use correct values.
 
- 1. Create a service account and download the key JSON file.
- https://cloud.google.com/docs/authentication/production#creating_a_service_account
-
- 2. Provide service account credentials using one of the following options:
- - set the GOOGLE_APPLICATION_CREDENTIALS environment variable, the API
- client will use the value of this variable to find the service account key
- JSON file.
-
- https://cloud.google.com/docs/authentication/production#setting_the_environment_variable
-
- OR
-
- - manually pass the path to the service account key JSON file to the API client
- by specifying the keyFilename parameter in the constructor.
- https://cloud.google.com/docs/authentication/production#passing_the_path_to_the_service_account_key_in_code
-
- To run this sample using Maven:
-   cd java-analytics-data/samples/snippets
-   mvn compile
-   mvn exec:java -Dexec.mainClass="QuickstartSample"
+To run this sample using Maven:
+  cd java-analytics-data/samples/snippets
+  mvn compile
+  mvn exec:java -Dexec.mainClass="com.example.analytics.QuickstartSample"
```

Comment:
Not the preferred way to do this according to the [Sample Format](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md) guide, but I won't ask for it to change.

Reply:
Well, at this point, perhaps we should match as it's been quite a while.